app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -27,7 +27,7 @@ def has_middleware_manager_servers(options = {})
       has_inventory({
         :model_class                 => provider_module::MiddlewareManager::MiddlewareServer,
         :association                 => :middleware_servers,
-        :inventory_object_attributes => %i(type_path hostname product lives_on_id lives_on_type
+        :inventory_object_attributes => %i(type type_path hostname product lives_on_id lives_on_type
```

Comment:
Is this how we do type for other objects?  cc @Ladas

Reply:
We do type like this if there are sub STI classes we need to set, which seems to be the case.